addons/stock/migrations/13.0.1.1/post-migration.py

```diff
@@ -175,8 +175,9 @@ def fill_stock_picking_type_sequence_code(env):
     openupgrade.logged_query(
         env.cr, """
         UPDATE stock_picking_type
-        SET sequence_code = CASE WHEN code = 'incoming' THEN 'IN'
-          WHEN code = 'outgoing' THEN 'OUT'
+        SET sequence_code = CASE WHEN code = 'outgoing' THEN 'OUT'
+          WHEN code = 'incoming' AND name LIKE '%Dropship%' THEN 'DS'
```

Comment:
Do this on its own module.

Reply:
sequence_code is not used on the rest of the script or code, as you shouldn't generate any new picking AFAIK.

Reply:
Done

Reply:
THIS SHOULD BE FIXED AGAIN :dancers:

Reply:
Yes, we should remove that replacing also from here, as we should respect the user preferences. Can we delay this change until merging this PR, or should we do it all in once?

Reply:
I think we should do it in the same PR.

Reply:
Done in 7be23b7. Is what you want?

Reply:
Yes :+1: